utilities/convert_frosst/convert_frcmod.py

```diff
@@ -90,6 +90,7 @@ def convert_frcmod_to_ffxml( infile, inxml, outxml ):
     force_section = ['NonbondedForce', 'HarmonicBondForce', 'HarmonicAngleForce', 'PeriodicTorsionForce', 'PeriodicTorsionForce']
     ct = 0
     thissec = None
+    # Why is this a while loop and not a for line in text loop?
```

Comment:
No good reason, other than that because I was going to be skipping lines with comments, coding this way came to mind first rather than using `for` with `continue` statements.